test/e2e/case33_empty_label_test.go

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) 2021 Red Hat, Inc.
+// Copyright Contributors to the Open Cluster Management project
+
+package e2e
+
+import (
+	"context"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"open-cluster-management.io/config-policy-controller/test/utils"
+)
+
+const (
+	case33ConfigPolicy     = "../resources/case33_empty_label_handling/empty-labels-config.yaml"
+	case33ConfigPolicyName = "case33-empty-labels"
+	case33NodeName         = "test-managed-control-plane"
```

Comment:
I'd prefer to use a different resource for the test. This relies on using a kind cluster of a specific name. If the test creates a configmap or something, the policy could just label that.